client/dhrystone.h

```diff
@@ -1,5 +1,5 @@
 
-#ifdef __APPLE__
```

Comment:
What exact warning is fixed by this?
Also, it's not related to this particular PR, so should be moved to another PR.

Reply:
Reverted back and resubmitted the change via a separate PR.